main.go

```diff
@@ -149,14 +157,16 @@ func (f *Flags) parseFlags(args []string) error {
 	flags.BoolVar(&f.TFE, "terraform-ent", false, "(Experimental) Run Terraform Enterprise diagnostics")
 	flags.BoolVar(&f.Vault, "vault", false, "Run Vault diagnostics")
 	flags.BoolVar(&f.AllProducts, "all", false, "DEPRECATED: Run all available product diagnostics")
+	flags.BoolVar(&f.Version, "version", false, "Print the current version of hcdiag")
+	flags.DurationVar(&f.IncludeSince, "include-since", SeventyTwoHours, "Alias for -since, will be overridden if -since is also provided, usage examples: `72h`, `25m`, `45s`, `120h1m90s`")
```

Comment:
Should we replace the backticks in the usage strings with single quotes based on what we discovered with the flag help showing the first backticked string as the type of input? Or, would a bug on the backlog be better for this?

Reply:
Definitely worth doing, I'm going to make a backlog issue for it so we can also ensure the input types/guides are all consistent and useful while we replace backticks too.